### PR TITLE
[Issue #45] Setting Screen

### PR DESCRIPTION
**Pull request by kozych**
Added setting activity with options
Enable/disable notifications for medicament reminders
Enable/disable notifications for medical appointments reminders
Enable/disable night mode
ask again for permissions

### CODE REVIEW

**Comment by piotrgos45** [comment]
In settings screen, the back arrow (on left top corner) doesn't work.

File: app/src/main/java/com/example/patientassistant/view/SettingsActivity.kt [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                            )
+                        }
+                    }
+                    true
+                }
+            }
+
+            nightMode?.setOnPreferenceChangeListener { preference, newValue ->
+                if (newValue == true) {
+                    AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES)
```
Comment (AlicjaKowalska) [inline app/src/main/java/com/example/patientassistant/view/SettingsActivity.kt:126]: Night mode doesn't work after destroying the app.

Comment (kozych) [reply app/src/main/java/com/example/patientassistant/view/SettingsActivity.kt:135]: done

**Comment by kozych** [comment]
> In settings screen, the back arrow (on left top corner) doesn't work.

done

**Review by AlicjaKowalska — APPROVED** [review]

**Review by piotrgos45 — APPROVED** [review]

**Review by michalmobi — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/patientassistant/view/SettingsActivity.kt [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                    Manifest.permission.ACCESS_FINE_LOCATION
+                ) == PackageManager.PERMISSION_GRANTED
+            ) {
+                locationPreference?.setTitle(R.string.locationPermissions_granted_title)
+                locationPreference?.isEnabled = false
+            } else {
+                locationPreference?.setOnPreferenceClickListener {
+                    when {
+                        ContextCompat.checkSelfPermission(
+                            settingsActivity,
```
[the remark this answers is not in the document]
Comment (kozych) [reply app/src/main/java/com/example/patientassistant/view/SettingsActivity.kt]: I think it is better now

**Review by michalmobi — APPROVED** [review]

**Pull request merged**